core/templates/pages/exploration-editor-page/modal-templates/metadata-version-history-modal.component.spec.ts

```diff
@@ -289,8 +289,8 @@ describe('Metadata version history modal', () => {
     tick();
 
     expect(alertsService.addWarning).toHaveBeenCalledWith(
-      'Could not fetch the version history data. ' +
-      'Please reload the page and try again.'
+      'Version history of cannot be explored further ' +
```

Comment:
Add a line to test for `ngbActiveModal`

Reply:
I have changed this part of code along with the tests.